docs/src/main/sphinx/release/release-366.md

```diff
@@ -0,0 +1,43 @@
+# Release 366 (XX Dec 2021)
+
+## General
+         
+* Add support for automatic query retries. This feature can be turned on by setting the `retry-policy` 
```

Comment:
Should we note this is not safe for DML queries for now?

Reply:
That should go in the documentation.